addons/rose/addon/styles/hds/overrides.scss

```diff
@@ -1,5 +1,6 @@
 // field bottom margin (HDS leaves this to implementers)
 [class^='hds-form-field'],
-[class^='hds-form-group'] {
+[class^='hds-form-group'],
+.hds-button {
```

Comment:
the buttons don't have a bottom margin like other HDS components. I could target only that particular `update filter` button in `app.scss`, but since it doesn't have a bottom margin at all, I thought it fit here